tests/next_tests/__init__.py

```diff
@@ -11,6 +11,77 @@
 # distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
+import enum
+
+import pytest
+
+
+# Skip definitions
+class SkipMark(enum.Enum):
+    XFAIL = pytest.xfail
+    SKIP = pytest.skip
+
+
+# Skip messages (available format keys: 'marker', 'backend')
+UNSUPPORTED_MESSAGE = "'{marker}' tests not supported by '{backend}' backend"
+BINDINGS_UNSUPPORTED_MESSAGE = "'{marker}' not supported by '{backend}' bindings"
+
+# Processors
+DACE = "dace_iterator.run_dace_iterator"
+GTFN_CPU = "otf_compile_executor.run_gtfn"
+GTFN_IMPERATIVE = "otf_compile_executor.run_gtfn_imperative"
+GTFN_CPU_WITH_TEMP = "otf_compile_executor.run_gtfn_with_temporaries"
+
+# Test markers
+USES_APPLIED_SHIFTS = "uses_applied_shifts"
+USES_ATLAS_TABLES = "uses_atlas_tables"
+USES_CAN_DEREF = "uses_can_deref"
+USES_CONSTANT_FIELDS = "uses_constant_fields"
+USES_DYNAMIC_OFFSETS = "uses_dynamic_offsets"
+USES_IF_STMTS = "uses_if_stmts"
+USES_INDEX_FIELDS = "uses_index_fields"
+USES_LIFT_EXPRESSIONS = "uses_lift_expressions"
+USES_NEGATIVE_MODULO = "uses_negative_modulo"
+USES_ORIGIN = "uses_origin"
+USES_REDUCTION_OVER_LIFT_EXPRESSIONS = "uses_reduction_over_lift_expressions"
+USES_SCAN_IN_FIELD_OPERATOR = "uses_scan_in_field_operator"
+USES_SPARSE_FIELDS = "uses_sparse_fields"
+USES_STRIDED_NEIGHBOR_OFFSET = "uses_strided_neighbor_offset"
+USES_TUPLE_ARGS = "uses_tuple_args"
+USES_TUPLE_RETURNS = "uses_tuple_returns"
+USES_ZERO_DIMENSIONAL_FIELDS = "uses_zero_dimensional_fields"
+
+# Skip matrix
```

Comment:
Just a question: what do you think about moving all those definitions to a different module (e.g. `exclusion_matrices.py`) to make them more visible to other developers? Would it make sense for you?

Reply:
Moving these definitions to a dedicated place is definitely more clear.